src-electron/db/query-config.js

```diff
@@ -211,6 +219,30 @@ async function insertOrUpdateAttributeState(
       attributeId,
       clusterRef
     )
+  if (
+    staticAttribute.storagePolicy ==
+    dbEnum.storagePolicy.attributeAccessInterface
+  ) {
+    staticAttribute.storagePolicy = dbEnum.storageOption.external
```

Comment:
Can we please not conflate storagePolicy and storageOption naming-wise?  They are different things.

Reply:
okay, thanks

Reply:
fixed, thank you